README.md

```diff
@@ -7,9 +7,8 @@
 
 ## Galaxy Europe
 
-- Use this anywhere we're really talking about the European Galaxy project as a collective.
+- Use this anywhere we're really talking about the European Galaxy project as a collective, e.g. posters next to your ELIXIR/de.NBI logos.
```

Comment:
I would not mention this. Most people do not know what de.NBI is and maybe even ELIXIR outside of Europe.